packages/cli-lib/lib/handleFieldsJs.js

```diff
@@ -172,8 +176,9 @@ function flattenArray(arr) {
 }
 
 //Transform fields array to JSON
-function fieldsArrayToJson(fields) {
-  fields = flattenArray(fields).map(field => {
+async function fieldsArrayToJson(fields) {
+  fields = await Promise.all(flattenArray(fields));
```

Comment:
Does this require a promise? `flattenArray` looks like it's synchronous

Reply:
`flattenArray` itself is synchronous, but the array it returns can have a mix of objects and Promises. The `await Promise.all` lets us wait for every Promise in the array to resolve before moving on.

Reply:
Ahh I see what you mean. Makes sense 👍